### PR TITLE
feat: added physics directory and ground to ground projectile motion algorithm

### PR DESCRIPTION
**Pull request by Focusucof**
#### Description of Change

Added a directory for physics related algorithms. Added a ground to ground projectile motion calculator algorithm.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 
The added file (in physics/ground_to_ground_projectile_motion.cpp) has functions to calculate the time of flight, the horizontal range, and the max height of a projectile launched from the ground.

### CODE REVIEW

File: physics/ground_to_ground_projectile_motion.cpp [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+/**
+ * @brief Main function
+ * @returns 0 on exit
+ */
+int main() {
+    test();  // run self-test implementations
+    return 0;
+}
```
Comment (Panquesito7) [inline physics/ground_to_ground_projectile_motion.cpp:133]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — COMMENTED** [review]
Great work! 🚀

**Review by Panquesito7 — APPROVED** [review]
Amazing work! Thank you. 🚀

**Review by mishraabhinn — APPROVED** [review]

**Comment by kvedala** [comment]
@mishraabhinn This pull request should not have been merged. There are changes requested.

**Comment by mishraabhinn** [comment]
> @mishraabhinn This pull request should not have been merged. There are changes requested.

I did not see any.

**Comment by kvedala** [comment]
@Focusucof Please recreate the pull-request with the suggested changes made.

**Pull request merged**